lib/lookdevXUsd/LookdevXUsd.cpp

```diff
@@ -117,6 +127,13 @@ void uninitialize()
         runTimeMgr.setSceneItemOpsHandler(mayaUsdRuntimeId, mayaUsdSceneItemOpsHandler);
     }
     mayaUsdSceneItemOpsHandler.reset();
+
+    // Unregister the decorated Maya handlers and restore the original ones.
+    Ufe::RunTimeMgr::instance().unregisterHandler(mayaRuntimeId, ProxyShapeLookdevHandler::id);
```

Comment:
This was the offending line. In the original PR it erroneously said `mayaUsdRuntimeId` instead of `mayaRuntimeId` i.e., it called unregister on the wrong runtime.

With that fixed, the offending test should now be passing.